imhotep/app.py

```diff
@@ -129,11 +136,22 @@ def invoke(self):
                     violating_lines)
                 for x in matching_numbers:
                     error_count += 1
+                    if error_count > max_errors:
+                        continue
                     reporter.report_line(
-                        repo.name, cinfo.origin, entry.result_filename,
+                        cinfo.origin, entry.result_filename,
                         x, pos_map[x], violations['%s' % x])
 
+                if error_count > max_errors \
```

Comment:
[Here](https://github.com/IMGGaming/imhotep/pull/1/files#diff-68ec65e900440102bcc609b7e15e6b41R99) on the method definition.

You can pass a `max_errors` parameter when invoking the linter:
* It is defaulted to infinite (no max error)
* The linter is invoked with no max error at the moment (see [main.py](https://github.com/IMGGaming/imhotep/blob/master/imhotep/main.py#L54))